packages/react/package.json

```diff
@@ -72,8 +72,7 @@
     "rollup-plugin-scss": "4.0.0",
     "rollup-plugin-typescript2": "0.34.1",
     "sass": "1.62.0",
-    "tslib": "2.5.0",
-    "typescript": "4.9.5"
```

Comment:
Removing typescript correct?

Reply:
[Typescript is defined in the root package.json](https://github.com/Amsterdam/design-system/blob/main/package.json#L47), I don't think we need to define it in all the sub packages, right?